mwdb/web/src/components/Settings/Views/GroupsList.js

```diff
@@ -20,15 +21,17 @@ function GroupItem(props) {
                 </Link>
             </td>
             <td>
-                {props.name === "public"
-                    ? "(Group is public and contains all members)"
-                    : props.users.map((login) => (
-                          <UserBadge
-                              user={{ login }}
-                              clickable
-                              basePath="/admin"
-                          />
-                      ))}
+                <LimitTo count={5}>
+                    {props.name === "public"
+                        ? "(Group is public and contains all members)"
+                        : props.users.map((login) => (
+                              <UserBadge
+                                  user={{ login }}
+                                  clickable
+                                  basePath="/admin"
+                              />
+                          ))}
+                </LimitTo>
```

Comment:
You will see "(Grou and 37 more" here
```suggestion
                {props.name === "public" ? (
                    "(Group is public and contains all members)"
                ) : (
                    <LimitTo count={5}>
                        {props.users.map((login) => (
                            <UserBadge
                                user={{ login }}
                                clickable
                                basePath="/admin"
                            />
                        ))}
                    </LimitTo>
                )}
```